.circleci/config.yml

```
@@ -13,33 +13,6 @@ references:
     key: python-env-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "requirements/requirements-test.txt" }}-{{ checksum "requirements/requirements-precommit.txt" }}-{{ checksum ".circleci/ci-oldest-reqs.txt" }}-{{ checksum ".pre-commit-config.yaml" }}
```

Let's remove the cache key's dependence on `requirements/requirements-precommit.txt` and `.pre-commit-config.yaml`. The same change will need to be applied on line 10 (I can't comment there because it's too far from the lines changed here).
```suggestion
    key: python-env-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "requirements/requirements-test.txt" }}-{{ checksum ".circleci/ci-oldest-reqs.txt" }}
```